docs/get-started.md

```diff
@@ -158,22 +158,22 @@ Currently, we are using `git-lfs` to store the 3rd-party libraries so make sure
 **Compiling**
```

Comment:
a tiny error in line 73: 
you're in the `root/` directory and you should use `cd ~/nebula-graph/`  to switch
you're in the root directory '/' and type `cd ~/nebula-graph/` to switch